src/core/Entry.cpp

```diff
@@ -632,6 +645,7 @@ Entry* Entry::clone(CloneFlags flags) const
         entry->m_uuid = m_uuid;
     }
     entry->m_data = m_data;
+    entry->m_customData->copyDataFrom(m_customData);
```

Comment:
Shouldn't we just call ```entry->copyDataFrom(this)``` for all these since we are duplicating code from below.

Reply:
Very good point. Fixed.